---
author:
  name: ferfolio_2
  picture: 126394
body: "I've completed the installation of FDK + the Macro installation successfully...\r\n\r\nBut
  when I go and run the \"Autohint\" macro (and some others too), I get:\r\n\"The
  script cannot run the command-line program 'autohintexe.exe'. Please make sure the
  AFDKO is installed, and the system environment variable PATH contains the path the
  to FDK sub-directory containing 'autohintexe.exe'.\"\r\n\r\nI run the Autohint in
  a command window and it worked, so the FDK is installed OK. I hae a 2.4 version
  of python, running on Windows 8 64bits.\r\n\r\nThen I tried to put the source folder
  in the macro, but it still throws some imposiblruuu output.\r\n\r\nAny ideas? I've
  cheked other typophile posts, and the adobe documentation, but no answer..."
comments:
- author:
    name: ferfolio_2
    picture: 126394
  body: "<strong>(i'm running python 2.4.4) The Autohint macro has a line that says</strong>\r\n<code>\r\nif
    os.name == \"nt\":\r\n\thaveAC = setFDKToolsPath(\"C:/FDK/Tools/win/autohintexe.exe\")\r\nelse:\r\n\thaveAC
    = setFDKToolsPath(\"autohintexe\")\r\n</code>\r\n\r\n<strong>Which I substituted
    for (just for trying):</strong>\r\n<code>\r\nif os.name == \"nt\":\r\n\thaveAC
    = setFDKToolsPath(\"C:/FDK/Tools/win/autohintexe.exe\")\r\nelse:\r\n\thaveAC =
    setFDKToolsPath(\"C:/FDK/Tools/win/autohintexe\")\r\n</code>\r\n\r\n\r\nAnd now
    the message is the following: \"Skipping glyph k. Failure in processing outline
    data Hinting l. 'autohintexe' is not recognized as an internal or external command,
    operable program or batch file.\"\r\n\r\nAt the end of the line:\r\n\"Log saved
    to  Traceback (most recent call last):\r\n  File \"<string>\", line 808, in ?\r\n
    \ File \"<string>\", line 790, in run\r\n  File \"<string>\", line 280, in doHinting\r\n
    \ File \"C:\\Program Files (x86)\\FontLab\\Studio5\\macros\\system\\modules\\AdobeFontLabUtils.py\",
    line 365, in close\r\n    print \"Log saved to \", self.fileName\r\n  File \"C:\\Python24\\lib\\codecs.py\",
    line 178, in write\r\n    data, consumed = self.encode(object, self.errors)\r\nUnicodeDecodeError:
    'ascii' codec can't decode byte 0xed in position 22: ordinal not in range(128)\"\r\n\r\n"
  created: '2013-04-27 17:50:22'
- author:
    name: ferfolio_2
    picture: 126394
  body: 'This is like advanced Chinese for me... '
  created: '2013-04-27 17:51:10'
- author:
    name: frankrolf
    picture: 116253
  body: "Did you also run the FinishInstallWindows.cmd file?\r\n\r\nQuoting the Read_Me_First.html
    file:\r\n<cite>Open the \"FDK\" folder. and find the file \"FinishInstallWindows.cmd\".\r\nOpen
    a command prompt window. If you do not know how to do this, read the file \"FDK\\Technical
    Documentation\\CommandLineHowTo.pdf\".\r\nDrag the icon for the file \"FinishInstallWindows.cmd\"
    onto the command prompt window. This pastes the path into the command prompt window.
    Press the \"enter\" key. This runs the program.\r\n</cite>\r\n\r\n"
  created: '2013-04-28 03:24:45'
date: '2013-04-27 17:17:18'
title: AFDKO 2.5 Question

---
